assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -177,6 +189,31 @@ const reducer = (state, action) => {
       loaded: false,
     };
   }
+  case ACTIONS.ADD_IMAGE_TO_POST: {
+    return {
+      ...state,
+      processing: true,
+    };
+  }
+  case ACTIONS.ADDED_IMAGE_TO_POST: {
+    return {
+      ...state,
+      processing: false,
+      imageAdded: true,
+    };
+  }
+  case ACTIONS.ADD_IMAGE_TO_POST_ERROR: {
```

Comment:
Just to be curious, is for any reason you create a new specific action error for this instead of using the [SET_ERROR](https://github.com/greenpeace/planet4-master-theme/pull/2112/files#diff-2d5f38a9e36d2b751a99282786e7adfec59d239898030fce76b89ddbbeb6a426R169)?

Reply:
Yes, your _SET_ERROR_ case does a lot of things which I am not fully aware of, plus I needed something simple I could track, and since the `error` state was available and not used in the `SideBar` component, I created a simple case to use that instead so I can also use it to display errors for the Media Archive.